setup.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.15.0
+current_version = 0.15.1-beta
```

Comment:
This has been a bit of a convention question for a while now. Essentially, sicne we are only now pushing stable (ie: no more beta-tagged) releases to PyPi and conda-forge, for the past few months, I've been asking that people bump the version by patch between formal releases. Since the next formal release that will have these changes will be 0.16.0 on PyPI/conda-forge I figure that this is fine and helps users who build off master know that they're using a changing/beta build if ever they have questions.

If we want, we can probably drop the `.x` on our history to make it less confusing.

Reply:
Want me to open a PR to implement the dropping of the `.x`?

Reply:
I think I understand, but if someone else makes a PR and bumps the version to 0.15.2, won't it get terribly confusing ?

Reply:
Unless they are producing a release, it would be bumped to `0.15.2-beta`. Once all the features in the milestone are met, I usually tag with `bump2version release` to drop the beta and push to PyPI. This has worked for us so far.

There have been a few instances where simultaneous branches have all bumped their versions up off the same master at the same time. I've made it so that master needs to be merged into the PR before it can be merged back to master for that reason. Bumping the version should be one of the last things performed.

I could make this a bit clearer in the developer documentation. This development pipeline kind of came to be over the past several months but it should be clear to everyone working on `xclim`. Gotta reduce that "bus factor".